tests/200_dbaas_policy.robot

```diff
@@ -8,200 +8,202 @@ Resource            ../resources/keywords/dbaas_policy.resource
 
 Scenario: Create DBaaSPolicy As Developer From Error screen on Admin View
-    [Tags]  dbaas-test-09
+    [Tags]    dbaas-test-09
     Given The User Logs In To Openshift As OCP_LDAP_DEV
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_DEV
     Then DBaaSPolicy Creation Forbidden
 
 Scenario: Create DBaaSPolicy As Project Admin From Error screen on Admin View And Import CrunchyDB
-    [Tags]  dbaas-test-10
+    [Tags]    dbaas-test-10
     Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_PROJECT_ADM
     Then User Is Able To Import CrunchyDB Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Import CrunchyDB
-    [Tags]  dbaas-test-11
+    [Tags]    dbaas-test-11
     Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
     Then User Is Able To Import CrunchyDB Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Project Admin From Error screen on Admin View And Import CockroachDB
-    [Tags]  dbaas-test-12
+    [Tags]    dbaas-test-12
     Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_PROJECT_ADM
     Then User Is Able To Import CockroachDB Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Import CockroachDB
-    [Tags]  dbaas-test-13
+    [Tags]    dbaas-test-13
     Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
     Then User Is Able To Import CockroachDB Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Project Admin From Error screen on Admin View And Import RDS
-    [Tags]  dbaas-test-14
+    [Tags]    dbaas-test-14
     Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_PROJECT_ADM
     Then User Is Able To Import RDS Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Import RDS
-    [Tags]  dbaas-test-15
+    [Tags]    dbaas-test-15
     Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
     Then User Is Able To Import RDS Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Project Admin From Admin view And Import MongoDB
```

Comment:
Please change the end to: 'Import MongoDB provider account' here and in the rest of the tests.
For consistency and for better understanding :-)

Reply:
@mrrajan IMHO, the scope of this PR is for formatting and I dont want to introduce any changes out of the scope. So, Shall I accommodate these suggestion on the next one?